cmd/controller-manager/app/controller_manager.go

```diff
@@ -300,7 +300,7 @@ func StartControllers(s *options.ControllerManagerServer,
 		}
 
 		glog.V(5).Info("Running controller")
-		go serviceCatalogController.Run(stop)
+		go serviceCatalogController.Run(int(s.ConcurrentSyncs), stop)
```

Comment:
I get it now the "controller-manager" vs the "controller". Weird split.

All comments on Run need to be updated to mention the new argument.